backend/browser.py

```diff
@@ -41,7 +41,24 @@ def _unzip_to_plugin_dir(self, zip, name, hash):
             return False
         zip_file = ZipFile(zip)
         zip_file.extractall(self.plugin_path)
-        Popen(["chown", "-R", "deck:deck", self.plugin_path])
+
+        USER = None
+        cmd = "who | awk '{print $1}' | sort | head -1"
+        # Get the user by checking for the first logged in user. As this is run
```

Comment:
Duplication of code. Please move to a utility method that can be invoked in browser and main.

Reply:
Done